torch/ao/quantization/observer.py

```diff
@@ -1092,15 +1092,17 @@ def _adjust_min_max(
         # Therefore, we avoid such division operation.
         downsample_rate = int(
```

Comment:
can you add some docs on the order of operations, when does this order work or does not work etc.?